stock_picking_portal/controllers/portal.py

```diff
@@ -0,0 +1,273 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import binascii
+from collections import OrderedDict
+
+from odoo import _, fields, http
+from odoo.exceptions import AccessDenied, AccessError, MissingError, UserError
+from odoo.http import request
+
+from odoo.addons.portal.controllers import portal
+from odoo.addons.portal.controllers.mail import _message_post_helper
+from odoo.addons.portal.controllers.portal import pager as portal_pager
+
+
+class CustomerPortal(portal.CustomerPortal):
+    def _get_prepared_operation_domain(self, partner):
+        """Returns a domain to search for operations for the given partner.
+
+        Args:
+            partner (res.partner): partner to search operations for.
+
+        Returns:
+            list: domain to search for operations for the given partner.
+
+        """
+        StockPicking = request.env["stock.picking"]
```

Comment:
For what do you create this variable? This variable use once.

Reply:
Thanks, I'll fix it